docs/metrics.md

```diff
@@ -0,0 +1,13 @@
+# Application Aware Quota metrics
+
+### kube_application_aware_resourcequota
```

Comment:
name should start with kubevirt since we create it, no? @sradco

Reply:
Done